### PR TITLE
add astral 1.6.1 recipe

### PR DESCRIPTION
**Pull request by hajapy**

### CODE REVIEW

File: recipes/astral/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab0c08f2467d35fcaeb7bad15274743d3ac1ad18b5391f64a0058a9cd192d37d
+
+build:
+  #noarch: python
```
Comment (synapticarbors) [inline recipes/astral/meta.yaml]: I think this should probably be a `noarch` package.

Comment (hajapy) [reply recipes/astral/meta.yaml]: Indeed! I commented it out to test if it was responsible for the failure, which we now know is due an upstream conda-build issue. I'll uncomment this and rebuild after that is fixed (and address any real build issues that may come up!)

**Comment by aachkar-miovision** [comment]
Thanks @marcelotrevisani for the commit ;)
@synapticarbors PR is ready now as noarch

**Comment by hajapy** [comment]
Thanks @marcelotrevisani for the commit ;)
@synapticarbors PR is ready now as noarch

**Comment by marcelotrevisani** [comment]
LGTM
I'm going to merge this. Thanks to put this in conda-forge

**Pull request merged**